x/staging/posts/simulation/decoder.go

```diff
@@ -21,12 +21,6 @@ func NewDecodeStore(cdc codec.Marshaler) func(kvA, kvB kv.Pair) string {
 			cdc.MustUnmarshalBinaryBare(kvB.Value, &postB)
 			return fmt.Sprintf("PostA: %s\nPostB: %s\n", postA.String(), postB.String())
 
-		case bytes.HasPrefix(kvA.Key, types.PostCommentsStorePrefix):
```

Comment:
I think we should have a decoder test for `CommentsStorePrefix` since it's still in use

Reply:
@bragaz  There are ids bytes from string of posts in the comment store. I think it is no need to decode with codec because the error or the panic will no happen.